src/flac/main.c

```diff
@@ -1278,98 +1278,109 @@ void show_help(void)
-	printf("  -q, --qlp-coeff-precision=#        Specify precision in bits\n");
-	printf("  -r, --rice-partition-order=[#,]#   Set [min,]max residual partition order\n");
+	printf("      --lax                          Allow encoder to generate non-Subset files\n");
 	printf("      --limit-min-bitrate            Limit minimum bitrate (for streaming)\n");
 	printf("  -j, --threads=#                    Set number of encoding threads\n");
-	printf("format options:\n");
-	printf("      --force-raw-format             Treat input or output as raw samples\n");
+	printf("      --ignore-chunk-sizes           Ignore data chunk sizes in WAVE/AIFF files\n");
+	printf("      --replay-gain                  Calculate ReplayGain & store in FLAC tags\n");
+	printf("      --cuesheet=FILENAME            Import cuesheet & store in CUESHEET block\n");
+	printf("      --picture=SPECIFICATION        Import picture & store in PICTURE block\n");
+	printf("  -T, --tag=FIELD=VALUE              Add a FLAC tag; may appear multiple times\n");
+	printf("      --tag-from-file=FIELD=FILENAME     Like --tag but gets value from file\n");
+	printf("  -S, --seekpoint={#|X|#x|#s}        Add seek point(s)\n");
+	printf("  -P, --padding=#                    Write a PADDING block of length # bytes\n");
+	printf("\n");
+	printf("Format options (encoding defaults to FLAC not OGG; decoding defaults to WAVE, \n");
+	printf("             chunks found by --keep-foreign-metadata-if-present will override):\n");
+#if FLAC__HAS_OGG
+	printf("      --ogg                          Use Ogg transport layer, output .oga\n");
+	printf("      --serial-number                Ogg serial number to assign (encoding)\n);
+	printf("                                     or to select for decoding\n);
+#endif
 	printf("      --force-aiff-format            Decode to AIFF format\n");
 	printf("      --force-rf64-format            Decode to RF64 format\n");
 	printf("      --force-wave64-format          Decode to Wave64 format\n");
 	printf("      --force-legacy-wave-format     Decode to legacy wave format\n");
 	printf("      --force-extensible-wave-format Decode to extensible wave format\n");
 	printf("      --force-aiff-c-none-format     Decode to AIFF-C NONE format\n");
 	printf("      --force-aiff-c-sowt-format     Decode to AIFF-C sowt format\n");
-	printf("raw format options:\n");
-	printf("      --endian={big|little}    Set byte order for samples\n");
-	printf("      --channels=#             Number of channels\n");
-	printf("      --bps=#                  Number of bits per sample\n");
-	printf("      --sample-rate=#          Sample rate in Hz\n");
-	printf("      --sign={signed|unsigned} Sign of samples\n");
-	printf("      --input-size=#           Size of the raw input in bytes\n");
-	printf("negative options:\n");
+	printf("      --force-raw-format             Treat input or output as raw samples\n");
+	printf("raw format options mandatory for encoding; --sign and --endian also for decoding\n");
```

Comment:
I think this is a little confusing. What do you think of this?
```suggestion
	printf("raw format options: (all options mandatory for encoding from raw input,\n");
	printf("                   --sign and --endian are mandatory for decoding to raw output)\n");
```